quadrat/sass/blocks/_post-comments.scss

```diff
@@ -142,3 +135,13 @@
 		}
 	}
 }
+
+// NOTE: This is a wp.com-specific fix so that the comment form presented there (highlander)
+// is NOT displayed as a GRID.
+body.highlander-enabled {
+	.wp-block-post-comments {
+		form {
+			display: revert;
+		}
+	}
+}
```

Comment:
I think we should keep the wpcom only css to a separate stylesheet like we have [here](https://github.com/Automattic/themes/blob/trunk/seedlet/inc/wpcom-style.css) on Seedlet. If I'm not mistaken, that doesn't get uploaded to dotorg when we bundle the theme on its zip and we keep that separation

Reply:
Beautiful.  I was wondering about that and why I commented as such.  I'll get those broken out.  Thanks @MaggieCabrera

Reply:
All moved.  And I learned how .com-specific additions work in the process.